modules/kubernetes-addons/variables.tf

```diff
@@ -22,43 +22,43 @@ variable "auto_scaling_group_names" {
 }
 
 variable "tags" {
+  description = "Additional tags (e.g. `map('BusinessUnit`,`XYZ`)"
   type        = map(string)
   default     = {}
-  description = "Additional tags (e.g. `map('BusinessUnit`,`XYZ`)"
 }
 
 variable "irsa_iam_role_path" {
+  description = "IAM role path for IRSA roles"
   type        = string
   default     = "/"
-  description = "IAM role path for IRSA roles"
 }
 
 variable "irsa_iam_permissions_boundary" {
+  description = "IAM permissions boundary for IRSA roles"
   type        = string
   default     = ""
-  description = "IAM permissions boundary for IRSA roles"
 }
 
 variable "eks_oidc_provider" {
-  type        = string
   description = "The OpenID Connect identity provider (issuer URL without leading `https://`)"
+  type        = string
 }
 
 variable "eks_cluster_endpoint" {
-  type        = string
   description = "Endpoint for your Kubernetes API server"
+  type        = string
 }
 
 variable "eks_cluster_version" {
-  type        = string
   description = "The Kubernetes version for the cluster"
+  type        = string
```

Comment:
I am setting these three default values to `null` in my PR. Once its merged then you can rebase from Main

Reply:
if you want, i can merge mine into yours instead of main

Reply:
Probably better to merge this PR first.